test/espnet2/enh/test_espnet_model.py

```diff
@@ -15,13 +15,20 @@
 from espnet2.enh.separator.transformer_separator import TransformerSeparator
 
 is_torch_1_2_plus = LooseVersion(torch.__version__) >= LooseVersion("1.2.0")
```

Comment:
You can drop this. 
We don't support torch 1.3.1>

Reply:
OK, I'll remove it.